app/serialization.py

```diff
@@ -171,13 +172,30 @@ def serialize_host(host, staleness_timestamps, for_mq=True, additional_fields=tu
     return serialized_host
 
 
+# get hosts not marked for deletion
+def _get_unculled_hosts(group):
+    from app import inventory_config
+
+    hosts = []
+    if len(group.hosts) > 0:
+        for host in group.hosts:
+            staleness_delta = datetime.now(tz=timezone.utc) - host.stale_timestamp
+            culling_delta = inventory_config().culling_culled_offset_delta
+
+            if staleness_delta < culling_delta:
+                hosts.append(host)
```

Comment:
The length check is redundant; the for-each loop will just skip if the length is 0.
Also, `culling_delta` is set to the same value on every iteration of the loop. That should either be moved above the loop, or just simplified since `culling_delta` is only used once (see below):
```suggestion
    for host in group.hosts:
        staleness_delta = datetime.now(tz=timezone.utc) - host.stale_timestamp
        if staleness_delta < inventory_config().culling_culled_offset_delta:
            hosts.append(host)
```

Reply:
Done!